pkgs/development/haskell-modules/configuration-darwin.nix

```diff
@@ -321,4 +321,7 @@ self: super: ({
   # same
   # https://hydra.nixos.org/build/174540882/nixlog/9
   jacinda = dontCheck super.jacinda;
+
+  # "resource exhausted (Too many open files)' (after 100 tests)"
```

Comment:
Have we ruled out that this is an intermittent problem on darwin builders. They have been very unreliable lately.

Reply:
Possibly, but I thought that was a valid reason to disable tests?